docs/toc.json

```diff
@@ -1,8 +1,6 @@
-[
-  {
+[{
```

Comment:
Do you have any formatters other than [Prettier](https://prettier.io) installed? This shouldn't be formatted this way.

Reply:
Oh yeah i didn't notice that. I think it because the automatic save feature in vscode that i not yet config it. I will fix the format now